content/agent/_index.md

```diff
@@ -71,19 +71,20 @@ Some options have their own set of flags and options detailed in a help message.
 
 ## Using the GUI
```

Comment:
The section title is `Using the GUI` but its missing the usage part. We should either:
- change the title to just `GUI` (similar to the CLI section)
- Or add information on how to use the GUI (or refer to an existing section which does that)

Reply:
Let's go with `GUI`